src/Scrabble/Search.hs

```diff
@@ -54,6 +56,22 @@ containsAll :: String -> Search
 containsAll s1 s2 = fst $ foldl' f (True, ups s2) (ups s1) where
   f (b,s) c = if elem c s then (b, delete c s) else (False,s)
 
+-- | Like the above, but if it encounters a character in in s1 not in
+--   s2, it deletes a blank and tries again before returning false
+containsAllWithBlanks :: String -> Search
```

Comment:
Interesting. So far, I think this file has been Scrabble agnostic. The functions that deal with blanks break that. I'm trying to think about what to do. Easy thing to do is just to not worry about it.